docker-compose.monitoring.yml

```
@@ -0,0 +1,457 @@
+  # Docker 시스템 정리 서비스 (OverlayFS 최적화)
+  docker-cleanup:
+    image: alpine:latest
+    container_name: docker-cleanup
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    command: |
+      sh -c '
+        echo "Starting Docker cleanup service..."
+        while true; do
+          echo "Running cleanup at $$(date)"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)